test/promise-component-mocha.js

```diff
@@ -118,6 +118,32 @@ describe('promise-component', function() {
             return onceData(component, 'outgoing', 'sentback', 'incoming', "hello");
         }).should.be.rejectedWith("hello world");
     });
+    it("should log rejections to console.error when no reject port attached", function() {
```

Comment:
I do not follow how you are forcing the reject port to not be attached.  That port is created inside promise-component, no?